metricbeat/module/docker/helper.go

```diff
@@ -35,37 +51,25 @@ func ExtractContainerName(names []string) string {
 	}
 	return strings.Trim(output, "/")
 }
+
 func BuildLabelArray(labels map[string]string) []common.MapStr {
 
-	output_labels := make([]common.MapStr, len(labels))
+	outputLabels := make([]common.MapStr, len(labels))
 	i := 0
 	var keys []string
 	for key := range labels {
 		keys = append(keys, key)
 	}
 	sort.Strings(keys)
 	for _, k := range keys {
+		// Replace all . in the labels by _
+		// TODO: WHY?
```

Comment:
I wasn't aware, that the above was and issue for values. I would expect it to be an issue for keys, but also values? Were the values defined as text or keywords? That could have been the issue, that if it was not defined as keywors that the tokenizer was applied? But this can be solved by applying the right mapping.

We should test the behaviour here, as in general I think it would be nice to keep the name of the labels the same as the user set it, as this is what he will probably look for.